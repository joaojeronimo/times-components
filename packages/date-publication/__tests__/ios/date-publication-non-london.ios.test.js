import shared from "../shared-non-london";

const dateBST = "2017-01-01T14:32:00.000Z";
const dateGMT = "2017-07-01T14:32:00.000Z";

shared(dateBST);
shared(dateGMT);
