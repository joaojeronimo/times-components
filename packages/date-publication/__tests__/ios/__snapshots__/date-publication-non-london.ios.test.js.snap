// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. date and times publication with non-london timezone 1`] = `"Sunday January 01 2017, 02:32pm GMT, The Times"`;

exports[`1. date and times publication with non-london timezone 2`] = `"Saturday July 01 2017, 03:32pm BST, The Times"`;
